rosidl_generator_java/package.xml

```diff
@@ -35,15 +35,18 @@
   <test_depend>rmw_implementation_cmake</test_depend>
   <test_depend>rosidl_generator_c</test_depend>
 
-  <test_depend>rosidl_parser</test_depend>
   <test_depend>rosidl_cmake</test_depend>
+  <test_depend>rosidl_parser</test_depend>
   <!-- Duplicated from rosidl_default_generator in order to avoid a circular dependency. -->
```

Comment:
I think most (if not all) of the rosidl dependencies below can be removed. I'll give it a try.

Reply:
@jacobperron were you able to remove the rest of the dependencies? Does it still work?

Reply:
It didn't seem to work. Needs more investigation, but leaving these dependencies here for now is okay.